sda-commons-server-kms-aws/build.gradle

```diff
@@ -1,5 +1,7 @@
 dependencies {
-  compile project(':sda-commons-server-starter')
+  compile project(':sda-commons-server-starter'), {
+    exclude group: 'io.openapitools.hal', module: 'swagger-hal'
+  }
```

Comment:
What is this dependency used for? I wondering a bit that something is depending on the old starter bundle